projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-related-posts.php

```diff
@@ -51,6 +51,27 @@ public function register_routes() {
 				},
 			)
 		);
+
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/(?P<id>[\d]+)',
+			array(
+				'args' => array(
+					'id' => array(
+						'description' => __( 'Unique identifier for the post.', 'jetpack' ),
+						'type'        => 'integer',
+					),
+				),
+				array(
+					'show_in_index'       => true,
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_related_posts' ),
+					'permission_callback' => function () {
+						return current_user_can( 'manage_options' );
```

Comment:
Should this be accessible to other registered users on the site as well?

Reply:
I changed the permission to `edit_post`. Does it make sense?